apps/web/src/styles/globals.css

```
@@ -52,10 +66,10 @@ input[type="number"] {
 
 #connect-button-wrapper * div,
 #connect-button-wrapper * button {
-  background: #000;
+  background: var(--connect-button-bg);
   border: 0;
   font-family: 'ptRoot', 'Arial', 'Helvetica', 'sans-serif' !important;
```

Stylelint (17.7.0)

[error] Expected no quotes around "ptRoot" (font-family-name-quotes)

(font-family-name-quotes)